spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/db/migration/sqlserver/MsSqlMigrateUriRegistrySqlCommand.java

```
@@ -33,6 +33,20 @@
 
 	@Override
 	protected void updateAppRegistration(JdbcTemplate jdbcTemplate, List<AppRegistrationMigrationData> data) {
+
+		// check and do same fix as in R__Hibernate_Sequence
+		// needed if we're coming from classic mode and skipper is not
+		// started first
+		boolean fixHibernateSequence = false;
+		try {
+			jdbcTemplate.execute("select 1 from hibernate_sequence");
+			fixHibernateSequence = true;
+		} catch (Exception e) {
```

Should we throw an exception or at least some error logging?